corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -24,6 +24,33 @@
 breadcrumbs todo
 {% endblock %}
 
+{% block js %}{{ block.super }}
+  <script type="text/html" id="datagridPages">
```

Comment:
Did you look at integrating our knockout pagination widget?

Reply:
the one from users, etc? it's too bundled up into a specific view structure TBH

Reply:
nm, I was thinking of something different. seeing if i can reuse

Reply:
OMG THANK YOU :heart: :heart: :heart: :heart: :heart: :heart:

Reply:
hmmm...there's a bit of a bug here. It appears to call the `goToPage` callback on each page change, also...the callback is erroneous the first time I change the page. Looking to see if it's something with my setup or the pagination code itself.

Reply:
Thank you for looking into integrating it 😀 Let me know if you want a second set of eyes on the bug.

Reply:
there were a few things. two pagination widgets don't play well together. also, I added functionality to load initial values for `currentPage` and `limit` based on specified get parameter slugs